# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2022, aiogram Team
# This file is distributed under the same license as the aiogram package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: aiogram \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-10-01 22:51+0300\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.10.3\n"

#: ../../api/types/passport_element_error_files.rst:3
msgid "PassportElementErrorFiles"
msgstr ""

#: aiogram.types.passport_element_error_files.PassportElementErrorFiles:1 of
msgid ""
"Represents an issue with a list of scans. The error is considered "
"resolved when the list of files containing the scans changes."
msgstr ""

#: aiogram.types.passport_element_error_files.PassportElementErrorFiles:3 of
msgid "Source: https://core.telegram.org/bots/api#passportelementerrorfiles"
msgstr ""

#: ../../docstring
#: aiogram.types.passport_element_error_files.PassportElementErrorFiles.source:1
#: of
msgid "Error source, must be *files*"
msgstr ""

#: ../../docstring
#: aiogram.types.passport_element_error_files.PassportElementErrorFiles.type:1
#: of
msgid ""
"The section of the user's Telegram Passport which has the issue, one of "
"'utility_bill', 'bank_statement', 'rental_agreement', "
"'passport_registration', 'temporary_registration'"
msgstr ""

#: ../../docstring
#: aiogram.types.passport_element_error_files.PassportElementErrorFiles.file_hashes:1
#: of
msgid "List of base64-encoded file hashes"
msgstr ""

#: ../../docstring
#: aiogram.types.passport_element_error_files.PassportElementErrorFiles.message:1
#: of
msgid "Error message"
msgstr ""
